### PR TITLE
➖ vuex

### LINKED ISSUE #4755: Replace vuex with pinia/useState
"Nuxt no longer provides a Vuex integration. Instead, the official Vue recommendation is to use pinia, which has built-in Nuxt support via a [Nuxt module](https://pinia.vuejs.org/ssr/nuxt.html)." https://nuxt.com/docs/migration/configuration#vuex

- pinia support nuxt bridge https://pinia.vuejs.org/ssr/nuxt.html
- useState composables https://nuxt.com/docs/getting-started/state-management

related code:
- https://github.com/search?q=repo%3Akodadot%2Fnft-gallery+getters.&type=code
- https://github.com/search?q=repo%3Akodadot%2Fnft-gallery+dispatch%28&type=code
- https://github.com/search?q=repo%3Akodadot%2Fnft-gallery+commit%28&type=code

### PR DESCRIPTION
**Pull request by roiLeo**
Close #4755 

Last piece #5979, 
we should move `import { ENDPOINTS } from '@kodadot1/vuex-options'` to ` @kodadot1/static`

### CODE REVIEW

File: pages/_prefix/transfer.vue [hunk 1]
```diff
@@ -380,23 +380,11 @@ export default class Transfer extends mixins(
         this.isLoading = false
         return
       }
-      const availableNodesByPrefix: { value: string }[] =
```
Comment (vikiival) [inline pages/_prefix/transfer.vue:383]: Use instapi for small hacking

**Review by vikiival — COMMENTED** [review]
Please reopen the issue that was asking to try different nodes on fail.

**Comment by yangwao** [comment]
this will be breaking thing, right?

**Comment by roiLeo** [comment]
7pkv9j

**Comment by yangwao** [comment]
@prury can check if anything is broken?

**Comment by prury** [comment]
> @prury can check if anything is broken? 

Yup, can do it on monday if that's not prob, I'm currently travelling and will be back on sunday

**Comment by prury** [comment]
- General site usability test - done. 

- this error appears when trying to mint a RMRK2 NFT:
image

- Failed NFT mint attempts:
https://kusama.subscan.io/tx/0xd8d92866dd813821dc3f74fa59b969ac87f56dea7c84efd552ee216d4f9adce9
https://kusama.subscan.io/tx/0x79176215c61c20f2e35cdd5853858e72bfdd282ab3a515293a5fe3379fc70201

- Snek Mint Was successful 
- Was not able to Mint for known BSX funds bug

- When trying to change my fee asset:
image

- lots of RPC errors if compared to beta: 
image

Transferred some assets:
https://basilisk.subscan.io/extrinsic/0xca553ee9ef904d3f82ed7ab263dd7cd75dc1e8ac125fa996b87b608fc2e324c4
Teleported some assets:
https://kusama.subscan.io/extrinsic/18425563-2

**Comment by roiLeo** [comment]
>     * this error appears when trying to mint a RMRK2 NFT:
>       image

I don't think it's related with this PR, have you tried on https://canary.kodadot.xyz/ ?

>     * Failed NFT mint attempts:
>       https://kusama.subscan.io/tx/0xd8d92866dd813821dc3f74fa59b969ac87f56dea7c84efd552ee216d4f9adce9
>       https://kusama.subscan.io/tx/0x79176215c61c20f2e35cdd5853858e72bfdd282ab3a515293a5fe3379fc70201

Tx look oki? Maybe not related with this PR, as I didn't touch minting process

>     * Snek Mint Was successful

yep wfm too

>     * Was not able to Mint for known BSX funds bug

wdym?

>     * When trying to change my fee asset:

wfm on snek

>     * lots of RPC errors if compared to beta:

Where? how did you manage to get this errors?


The only thing that changing here are the fallback on transfer page if current node is off

**Comment by prury** [comment]
> > this error appears when trying to mint a RMRK2 NFT:
> 
> I don't think it's related with this PR, have you tried on https://canary.kodadot.xyz/ ?
> 
> > Failed NFT mint attempts:
> > https://kusama.subscan.io/tx/0xd8d92866dd813821dc3f74fa59b969ac87f56dea7c84efd552ee216d4f9adce9
> > https://kusama.subscan.io/tx/0x79176215c61c20f2e35cdd5853858e72bfdd282ab3a515293a5fe3379fc70201
> 
> Tx look oki? Maybe not related with this PR, as I didn't touch minting process

ugh, not working there as well, srry

> 
> > Snek Mint Was successful
> 
> yep wfm too

nice

> 
> > Was not able to Mint for known BSX funds bug
> 
> wdym?

not related to this PR : #6134 

> 
> > When trying to change my fee asset:
> 
> wfm on snek

can you also try on BSX?  because its working well on canary
> 
> > lots of RPC errors if compared to beta:
> 
> Where? how did you manage to get this errors?
> 
> The only thing that changing here are the fallback on transfer page if current node is off

it was exactly on transfers

**Comment by roiLeo** [comment]
> can you also try on BSX? because its working well on canary

`Inability to pay some fees , e.g. account balance too low` on both for me

> > > lots of RPC errors if compared to beta:
> > 
> > 
> > Where? how did you manage to get this errors?
> > The only thing that changing here are the fallback on transfer page if current node is off
> 
> it was exactly on transfers

any idea how to reproduce it? which chain?

**Comment by vikiival** [comment]
> Inability to pay some fees , e.g. account balance too low

Simple the user does not have balance to pay fees

**Comment by vikiival** [comment]
> Tx look oki? Maybe not related with this PR, as I didn't touch minting process

Tx is wrong, nothing was minted. Bear in mind that RMRK is string in blockchain

**Comment by prury** [comment]
> > Inability to pay some fees , e.g. account balance too low
> 
> Simple the user does not have balance to pay fees

but i do have balance tho, not too much but i have, i even disabled carbon less and some other option on settings

image

**Comment by prury** [comment]
> > can you also try on BSX? because its working well on canary
> 
> `Inability to pay some fees , e.g. account balance too low` on both for me
> 
> > > > lots of RPC errors if compared to beta:
> > > 
> > > 
> > > Where? how did you manage to get this errors?
> > > The only thing that changing here are the fallback on transfer page if current node is off
> > 
> > 
> > it was exactly on transfers
> 
> any idea how to reproduce it? which chain?

forget it, also happening on canary, but is on RMRK2

**Comment by roiLeo** [comment]
> forget it, also happening on canary, but is on RMRK2

Transfer [worked](https://kusama.subscan.io/extrinsic/0xee60591ad7de879730348b2f4b2004c7d3dc7d2acde8b2a18dd0581440e84e01) for me on rmrk2

**Comment by yangwao** [comment]
can't be bad idea

**Comment by roiLeo** [comment]
🎉 🥳 🎈

**Pull request merged**